prime-router/src/testIntegration/kotlin/datatests/mappinginventory/xcn/XCNtoPractitionerTests.kt

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.prime.router.datatests.mappinginventory.xcn
+
+import gov.cdc.prime.router.datatests.mappinginventory.verifyHL7ToFHIRToHL7Mapping
+import org.junit.jupiter.api.Test
+
+class XCNtoPractitionerTests {
+    @Test
+    fun `test translate to HL7 to FHIR to HL7`() {
```

Comment:
Nit: could we split this up into separate tests?  I think it makes it slightly better as documentation of how we expect things to work